.gitignore

```
@@ -0,0 +1 @@
+**/Manifest.toml
```

Isn't
```suggestion
Manifest.toml
```
sufficient here?